### PR TITLE
feat: added docker dev and prod files

### PR DESCRIPTION
**Pull request by TarunTomar122**

### CODE REVIEW

File: .docker/dev/client.dev.Dockerfile
Comment (kunxl-gg) [inline .docker/dev/client.dev.Dockerfile:1]: Does this reload whenever the client code is changed. I just see this building once, and serving the build files.

**Review by kunxl-gg — CHANGES_REQUESTED** [review]

File: .docker/dev/client.dev.Dockerfile
Comment (TarunTomar122) [reply .docker/dev/client.dev.Dockerfile:1]: nope. I just copied the original docker file as a placeholder. but with that being said I feel I should do something abt the reloading part at some point

**Review by rohansingh9001 — APPROVED** [review]
I trustie my Bestie.

File: .docker/dev/client.dev.Dockerfile
Comment (TarunTomar122) [reply .docker/dev/client.dev.Dockerfile:1]: I'll create an issue for this cz I couldn't make it work myself🥇

**Pull request merged**